src/QuickMarcEditor/utils.js

```diff
@@ -452,7 +452,7 @@ export const removeFieldsForDuplicate = (formValues) => {
 
 const checkIsEmptyContent = (field) => {
   if (typeof field.content === 'string') {
-    return compact(field.content.split(' ')).every(content => /^\$[a-z0-9]*/.test(content));
+    return compact(field.content.split(' ')).every(content => /^\$[a-z0-9]?$/.test(content));
```

Comment:
Please add a couple of tests for this - check that fields with empty content are removed and content like `$atesting $btestingtwo` is not considered to be empty